### PR TITLE
workflows: extract workspace-root-path from a new top section from .yaml file

### PR DESCRIPTION
**Pull request by marcdiazsan**
It passes a workspace_root_path as a parameter to rwc

Closes reanahub/reana-client#545

### CODE REVIEW

File: reana_server/rest/workflows.py [hunk 1]
```diff
@@ -401,10 +401,15 @@ def create_workflow(user):  # noqa
         workflow_dict["operational_options"] = validate_operational_options(
             workflow_engine, reana_spec_file.get("inputs", {}).get("options", {})
         )
+        workspace_root_path = reana_spec_file.get("workspace", {}).get(
+            "workspace_root_path", None
```
Comment (mvidalgarcia) [inline reana_server/rest/workflows.py:405]: Do we have to call it `workspace_root_path`? Isn't it a bit redundant having `workspace` at the top level?
IMO we could simplify it and use just `path` or `root_path`. What do you think?

```yaml
...
workspace:
  path: /eos/home-s/simko/myworkflows
...
```

BTW, is it definite to have `workspace` at the top level? I thought we were discussing it [here](https://github.com/reanahub/reana-client/issues/545).

PS. For `get()`, you don't need to specify `None` as it's the default value.

Comment (mvidalgarcia) [reply reana_server/rest/workflows.py:405]: cc/ @tiborsimko @audrium

Comment (marcdiazsan) [reply reana_server/rest/workflows.py:405]: I used `workspace_root_path` as it is not the full `workspace_path`, the full path would also have the `workflow_uuid`. So I agree that `root_path` would be a good simplification

Comment (audrium) [reply reana_server/rest/workflows.py]: Looks good, +1 for `workspace_path`.
Also, @marcdiazsan,  for `get()`, you don't need to specify `None` as @mvidalgarcia mentioned

Comment (marcdiazsan) [reply reana_server/rest/workflows.py]: Yes this one can be closed as is included into the other PR

**Comment by audrium** [comment]
Closing since all the commits from this PR are included in [another one](https://github.com/reanahub/reana-server/pull/397)

**Pull request closed without merging**